### PR TITLE
chore: added time-travel dungeon

### PR DESCRIPTION
**Pull request by hkumar0132**

### CODE REVIEW

File: english/time-travel/1/0.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+**Future is pretty exciting. Flying cars all around but global warming has exploited almost all of earth's resources.** 
+
+*SpaceX and NASA are working together to send everyone to Mars since Earth has become inhabitable*
+
+- [😥 You get overwhelmed and go back to the past](../begin-journey.md) 
+- [😎 Stay and help people colonize on Mars](1.md)
+
+
+
+
```
Comment (ming-tsai) [inline english/time-travel/1/0.md:10]: Delete these lines

**Review by ming-tsai — CHANGES_REQUESTED** [review]

File: english/time-travel/1/0.md (hunk 1 above)
Comment (hkumar0132) [reply english/time-travel/1/0.md]: Done.

**Review by ming-tsai — APPROVED** [review]

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @hkumar0132,
Could you add your dungeon on this [options](https://github.com/hkumar0132/markdown-dungeon/blob/add-dungeon/start-new-game.md)

File: src/pages/index.jsx [hunk 2]
```diff
@@ -46,9 +46,10 @@ export default function Home({ data }) {
         <Container className={classes.cardGrid} maxWidth='lg'>
           {/* End hero unit */}
           <Grid container spacing={4}>
-            {posts.map((post, index) => (
-              <CardGrid item key={index} post={post} xs={12} sm={6} md={3} />
-            ))}
+            {posts.map((post, index) => {
+              console.log(post, index);
+              return <CardGrid item key={index} post={post} xs={12} sm={6} md={3} />
+            })}
```
Comment (ming-tsai) [inline src/pages/index.jsx:52]: Rollback this change and `package-lock.json`

**Comment by hkumar0132** [comment]
Hey @ming-tsai I have made the changes. Let me know if any more changes required.

**Review by ming-tsai — APPROVED** [review]
Nice work @hkumar0132 🎉🚀🥇

**Pull request merged**